src/display/network.js

```diff
@@ -53,14 +53,7 @@ class NetworkManager {
   }
 
   requestRange(begin, end, listeners) {
-    const args = {
-      begin,
-      end,
-    };
-    for (const prop in listeners) {
-      args[prop] = listeners[prop];
-    }
-    return this.request(args);
+    return this.request({ begin, end, ...listeners });
```

Comment:
Can't we just have `requestRange(options) { return this.request(optinos); }` ?

Reply:
This comment made me take another look at the code, so I've just submitted PR #19273 instead :-)